HackerRank Problem/Python/Pairs.py

```diff
@@ -0,0 +1,76 @@
+'''
+You will be given an array of integers and a target value. Determine the number of pairs of array elements that have a 
+difference equal to a target value.
+
+For example, given an array of [1, 2, 3, 4] and a target value of 1, we have three values meeting the condition: , , and .
+
+Function Description
+
+Complete the pairs function below. It must return an integer representing the number of element pairs having the required 
+difference.
+
+pairs has the following parameter(s):
+
+k: an integer, the target difference
+arr: an array of integers
+Input Format
+
+The first line contains two space-separated integers  and , the size of  and the target value.
+The second line contains  space-separated integers of the array .
+
+Constraints
+
+each integer  will be unique
+Output Format
+
+An integer representing the number of pairs of integers whose difference is .
+
+Sample Input
+
+5 2  
+1 5 3 4 2  
+Sample Output
+
+3
+Explanation
+
+There are 3 pairs of integers in the set with a difference of 2: [5,3], [4,2] and [3,1] .
+'''
+
+#!/bin/python
+
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the pairs function below.
+def pairs(k, arr):
+    d = {}
+    answer = 0
+    for i in arr:
+        d[i] = i
+    for j in arr:
+        g = j+k
+        if g in d:
+            answer +=1
+    return answer
+if __name__ == '__main__':
+    fptr = open(os.environ['OUTPUT_PATH'], 'w')
+
+    nk = raw_input().split()
+
+    n = int(nk[0])
+
+    k = int(nk[1])
+
+    arr = map(int, raw_input().rstrip().split())
+
```

Comment:
please remove the unnecessary extra lines in your code

Reply:
Should i remove the problem statement too?

Reply:
No, Its fine

Reply:
delete the extra lines for example line number 58, 60, 62 and so on...